backend/sample/test.json

```diff
@@ -59,7 +77,10 @@
                 "en" : ["Passion fruit yogurts"],
                 "fr" : ["yaourts au fruit de la passion"]
             },
-            "tagsid" : ["en:Passion fruit yogurts", "fr:yaourts au fruit de la passion"],
+            "tagsid" : {
+                "en" : ["Passion fruit yogurts"],
+                "fr" : ["yaourts au fruit de la passion"]
+            },

```

Comment:
In tagsid you have to normalize spaces and remove stopwords for the language.

I let you do it in the other entries ;-)

```suggestion
            "tagsid" : {
                "en" : ["passion-fruit-yogurts"],
                "fr" : ["yaourts-fruit-passion"]
            },

```

Reply:
Oh okay, I'll check which entries have stopwords and normalize them.